### DISCUSSION TITLE
compile spark connector fail

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#9788] [A - General / Q&A] [answered]
i flow the doc : https://doris.apache.org/zh-CN/ecosystem/spark-doris-connector.html#%E7%89%88%E6%9C%AC%E5%85%BC%E5%AE%B9
to compile the spark connector  
i can throught pass the 3.1.2-2.12-1.0.0  tag 

but, because of  i used spark 3.2.1 version, so i compile the master branch  
execute comand :   

```shell
alternatives --set java java-1.8.0-openjdk.x86_64
alternatives --set javac java-1.8.0-openjdk.x86_64
export JAVA_HOME=/usr/lib/jvm/java-1.8.0

sh build.sh --spark 3.2.1 --scala 2.12 \
--mvn-args "-Dnetty.version=4.1.68.Final -Dfasterxml.jackson.version=2.12.3"  
```

exception:

![image](https://user-images.githubusercontent.com/11519151/170426354-0cbc0ebc-7831-4db7-b827-7194f70a3391.png)

**Comment by LOVEGISER** [comment, marked as answer]
now the spark-doris-connecter  is  support to  spark3.2, spark3.2.1 not support,we will support spark3.2.1 quickly in next commit.

**Comment by LOVEGISER** [comment]
doris-spark-conector  now have support  spark3.2.1
the bugs fix in  pr [  https://github.com/apache/incubator-doris-spark-connector/pull/33  ]

**Reply by young138120** [reply]
3Q
